doc/reference/network_bridge.md

```diff
@@ -6,39 +6,50 @@ discourse: 7322
-DHCP server and if desired also perform NAT for the bridge (this is the default.)
+The `bridge` network type allows to create an L2 bridge that connects the instances that use it together into a single network L2 segment.
+Bridges created by LXD are managed, which means that in addition to creating the bridge interface itself, LXD also sets up a local `dnsmasq` process to provide DHCP, IPv6 route announcements and DNS services to the network.
+By default, it also performs NAT for the bridge.
 
-When a bridge is managed by LXD, configuration values under the `bridge` namespace can be used to configure it.
+See {ref}`network-bridge-firewalld` for instructions on how to configure firewalld to work with LXD bridge networks.
 
-```{toctree}
-:maxdepth: 1
+## IPv6 prefix size
 
-Integrate with systemd-resolved </howto/network_bridge_resolved>
-Configure Firewalld </howto/network_bridge_firewalld>
-```
+If you're using IPv6 for your bridge network, you should use a prefix size of 64.
+
+Larger subnets (i.e., using a prefix smaller than 64) should work properly too, but they aren't typically that useful for {abbr}`SLAAC (Stateless Address Auto-configuration)`.
+
+Smaller subnets are in theory possible (when using stateful DHCPv6 for IPv6 allocation), but they aren't properly supported by `dnsmasq` and might cause problems.
+If you must create a smaller subnet, use static allocation or another standalone router advertisement daemon.
 
 (network-bridge-options)=
 ## Configuration options
 
-A complete list of configuration settings for LXD networks can be found below.
-
-The following configuration key namespaces are currently supported for bridge networks:
+The following configuration key namespaces are currently supported for the `bridge` network type:
 
+ - `bgp` (BGP peer configuration)
  - `bridge` (L2 interface configuration)
+ - `dns` (DNS server and resolution configuration)
  - `fan` (configuration specific to the Ubuntu FAN overlay)
- - `tunnel` (cross-host tunneling configuration)
  - `ipv4` (L3 IPv4 configuration)
  - `ipv6` (L3 IPv6 configuration)
- - `dns` (DNS server and resolution configuration)
+ - `maas` (MAAS network identification)
+ - `security` (network ACL configuration)
  - `raw` (raw configuration file content)
+ - `tunnel` (cross-host tunneling configuration)
+ - `user` (free-form key/value for user metadata)
 
-It is expected that IP addresses and subnets are given using CIDR notation (`1.1.1.1/24` or `fd80:1234::1/64`).
+```{note}
+{{note_ip_addresses_CIDR}}
+An exception are tunnel local and remote addresses.
+These should be specified as plain addresses (`192.0.2.0` or `2001:db8::`).
+```
 
-The exception being tunnel local and remote addresses which are just plain addresses (`1.1.1.1` or `fd80:1234::1`).
+The following configuration options are available for the `bridge` network type:
 
 Key                                  | Type      | Condition             | Default                   | Description
```

Comment:
The kind of small consistency edits as seen in the table here, could do with their own commit as that requires less active review (it's a mostly mechanical change).